lib/pom.rb

```diff
@@ -15,7 +15,7 @@ def log(message=nil)
 end
 
 default_gems = [
-    ['bundler', '1.17.3'],
+    # Bundler is shipped with RubyGems
```

Comment:
We need to keep Bundler installing as a default gem or it will not be upgradable by users.